src/pages/requests/users/external-from-main/[requestId]/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { ExternalFromMainOrgUserCreationRequestPage } from '../../../../../components/ExternalFromMainOrgUserCreationRequestPage/ExternalFromMainOrgUserCreationRequestPage';
+import { pages } from '../../../../../hooks/useRouter';
+import { trpc } from '../../../../../trpc/trpcClient';
+import { createGetServerSideProps } from '../../../../../utils/createGetSSRProps';
+
+export const getServerSideProps = createGetServerSideProps({
+    requireSession: true,
+    stringIds: { requestId: true },
+    action: async ({ stringIds, session, ssg }) => {
+        if (!session.user.role?.createUser || !session.user.role.editUserCreationRequests) {
+            return {
+                redirect: {
+                    destination: pages.home,
+                    permanent: false,
+                },
+            };
+        }
+        await ssg.userCreationRequest.getById.fetch(stringIds.requestId);
+
+        return { requestId: stringIds.requestId };
+    },
+});
+
+export default function ExternalFromMainOrgUserCreationRequest({ requestId }: { requestId: string }) {
+    const requestFormQuery = trpc.userCreationRequest.getRequestForExternalFromMainEmployeeById.useQuery(requestId);
+
+    const requestQuery = trpc.userCreationRequest.getById.useQuery(requestId);
+
+    if (!requestFormQuery.data) return null;
+
+    return (
+        <ExternalFromMainOrgUserCreationRequestPage
+            requestId={requestId}
```

Comment:
I guess you have `requestId` inside `request`

Reply:
No its not, maybe I should rename it to requestForm or something